akka-stream/src/main/boilerplate/akka/stream/scaladsl/ZipWithApply.scala.template

```diff
@@ -56,17 +65,13 @@ class ZipWith1[[#A1#], O] (zipper: ([#A1#]) ⇒ O) extends GraphStage[FanInShape
 
     setHandler(out, new OutHandler {
       override def onPull(): Unit = {
-        pending = shape.inlets.size
-        if (willShutDown) completeStage()
-        else {
-            [#pull(in0)#
-            ]
-        }
+        pending += shape.inlets.size
+        if (pending == ##0) pushAll()
       }
     })
   }
 
-  override def toString = "Zip"
+  override def toString = "ZipWith1"
```

Comment:
Why? We want that number to be ZipWith2, ZipWith3, ZipWith4 etc.
`##0` hardcodes it to 0.